.github/workflows/ci-github-actions.yaml

```diff
@@ -40,32 +40,32 @@ jobs:
         include:
           - jobname: GCC9-NoMPI-Debug-Real
             container:
-              image: williamfgc/qmcpack-ci:ubuntu20-openmpi
+              image: walshmm/qmcpack-ci:ubuntu22-openmpi
```

Comment:
@prckent time for a QMCPACK DockerHub account?

Reply:
Investigation can be done after upgrading the CI here.

Reply:
Hopefully it's just redirecting the container host service, @walshmm thoughts?

Reply:
It looks to operate just like dockerhub except you login to a github url through cli after providing a personal token.
```
export CR_PAT=YOUR_TOKEN
echo $CR_PAT | docker login ghcr.io -u USERNAME --password-stdin
```

assuming a user has the right perms, this could be what we're looking for.
```
docker push ghcr.io/NAMESPACE/IMAGE_NAME:latest
```
(where namespace would be QMCPACK)